php/src/Google/Protobuf/Internal/EnumDescriptorProto_EnumReservedRange.php

```diff
@@ -36,9 +36,21 @@ class EnumDescriptorProto_EnumReservedRange extends \Google\Protobuf\Internal\Me
     private $end = 0;
     private $has_end = false;
 
-    public function __construct() {
+    /**
+     * Constructor.
+     *
+     * @param array $data {
+     *     Optional. Data for populating the Message object.
+     *
+     *     @type int $start
+     *           Inclusive.
```

Comment:
Should we not print the comments after the fields? These don't really fit in this case, and I imagine in most cases where this is done. An example of this:

```proto
  message ReservedRange {
    optional int32 start = 1; // Inclusive.
    optional int32 end = 2;   // Exclusive.
  }
```